Pod/Classes/LSSpan.m

```diff
@@ -114,6 +114,46 @@ - (void)log:(NSString*)eventName
     [self _appendLog:logRecord];
 }
 
+- (void)log:(NSDictionary<NSString*, NSObject*>*)fields {
+    [self log:fields timestamp:[NSDate date]];
+}
+
+- (void)log:(NSDictionary<NSString*, NSObject*>*)fields timestamp:(nullable NSDate*)timestamp {
+    // No locking is required as all the member variables used below are immutable
+    // after initialization.
+    if (![m_tracer enabled]) {
+        return;
+    }
+    
+    LSPBLog* logRecord = [[LSPBLog alloc] init];
+    logRecord.timestamp = [LSUtil protoTimestampFromDate:timestamp];
+    NSMutableArray<LSPBKeyValue*>* logKeyValues = [NSMutableArray<LSPBKeyValue*> array];
```

Comment:
tiny nit: using `initWithCapacity` with `[fields count]` might be good here on the off-chance the log payload was really large